app/models/setting.rb

```diff
@@ -10,15 +10,17 @@ class Setting < ActiveRecord::Base
             :event_end_date,
             :event_url,
             :available_spots,
+            :required_rates_num,
+            :confirmation_time,
             presence: true
-  validates :available_spots, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :required_rates_num, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
+  validates :available_spots, :required_rates_num, :confirmation_time,
+            numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
   def self.get
     self.first || self.create({
       available_spots: 0,
       required_rates_num: 3,
+      confirmation_time: 7,
```

Comment:
Well I can't store 7.days in the db and I don't think mapping 7.days (model) to 7 (db) is a good idea. Good naming should be enough, I'll change it to what you proposed.